packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTable/__tests__/MyProfileCounterpartiesTable.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { p2p } from '@deriv/api';
+import MyProfileCounterpartiesTable from '../MyProfileCounterpartiesTable';
+
+const mockProps = {
+    dropdownValue: 'all',
+    setShowHeader: jest.fn(),
+    searchValue: '',
+};
+
+const mockApiValues = {
+    data: [],
+    isFetching: false,
+    isLoading: false,
+    loadMoreAdvertisers: jest.fn(),
+};
+
+jest.mock('@deriv/api', () => ({
+    p2p: {
+        advertiser: {
+            useGetList: jest.fn(() => mockApiValues),
+        },
+    },
+}));
+
+jest.mock('../../../../../../components/Modals/BlockUnblockUserModal', () => ({
```

Comment:
now that we have the absolute paths for imports, can you try using it here?
```suggestion
jest.mock('@/components/Modals', () => ({
```